etcdctlv3/command/util.go

```diff
@@ -47,3 +50,9 @@ func argify(s string) []string {
 	r := regexp.MustCompile("'.+'|\".+\"|\\S+")
 	return r.FindAllString(s, -1)
 }
+
+func contextWithTotalTimeout(cmd *cobra.Command) (context.Context, context.CancelFunc) {
+	var timeOut time.Duration
+	cmd.Flags().DurationVar(&timeOut, "total-timeout", 0, "")
```

Comment:
this is not total-timeout. this is request timeout. we probably need another dial timeout for dial the connection (we dial in newClient func in global.go). yes, we can make a total-timeout but this is not.